src/js/backend.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+(function () {
+  var URL = 'http://localhost:8888/code-and-magick/data';
+
+  window.upload = function (data, onSuccess) {
+    var xhr = new XMLHttpRequest();
+    xhr.responseType = 'json';
+
+    xhr.addEventListener('load', function () {
+      onSuccess(xhr.response);
+    });
+
+    xhr.open('POST', URL);
+    xhr.send(data);
+  };
+
+  window.load = function (onSuccess) {
```

Comment:
```suggestion
  window.load = function (onSuccess, onError) {
```

![2021-03-22_15-24-14](https://user-images.githubusercontent.com/19177308/111989477-deeaa280-8b22-11eb-94da-1ad2f6035a90.png)

скорее всего логику с показом сообщения об ошибке нужно перенести в этот коллбэк